### ISSUE TITLE
Document hide/show test changes.sh

### ISSUE THREAD

**Issue by cmungall** [linkml/linkml#602]
* https://github.com/linkml/linkml/blob/main/show_test_changes.sh
* https://github.com/linkml/linkml/blob/main/hide_test_changes.sh
* https://github.com/linkml/linkml/blob/main/checkout_outputs.sh

these are not documented

can't find any record
https://github.com/search?q=%22show_test_changes.sh%22

@hsolbrig says he has documentation for these, he will find and add to the test docs

**Comment by dalito** [comment]
As someone being new to linkml I found working with the testsuite difficult.

* Default test runs are very long (10min for testing one Python version)
* Running the tests changes many files in `tests/**/output/`-directories and these are not ignored via `.gitignore`

The latter is not nice if linkml is installed in developer mode `pip install -e .` and the test are executed locally (I used pytest). After running the tests (without committing the changes made) all the test-induced changes show up. I had >2300 modified files.

`contributing.md` does say anything about this. I wonder if you always run tests in an isolated env via tox?
